modules/navigation/nav_map.cpp

```diff
@@ -164,13 +161,14 @@ Vector<Vector3> NavMap::get_path(Vector3 p_origin, Vector3 p_destination, bool p
 					continue;
 				}
 
+				gd::NavigationPoly *least_cost_poly = &navigation_polys[least_cost_id];
```

Comment:
Nit: Could move this up next to `&edge`, no reason to grab this reference on every connection.  Also, could this be a reference instead of a pointer?

Reply:
There is a `push_back()` at the end of each connection, so the reference has to be updated.

I'll change the pointer to a reference.